advice_animal/cli.py

```diff
@@ -59,41 +61,70 @@ class Settings:
     help=f"""\
 Set the repo url from which advice is dispensed.  This is typically set by
 wrappers, the ADVICE_URL env var, or has a default of {ADVICE_SAMPLE_URL} worst
-case.  (Click's API prevents showing the actual live url here.)
+case.  (Click's API prevents showing the actual live url here, but check --config)
 """,
 )
 @click.option(
     "--skip-update",
     is_flag=True,
     help="When loading advice from a url, don't pull if some version already exists locally.",
 )
+# Filtering
 @click.option(
     "--confidence", default="unset", help="Filter advice to be at least this confident"
 )
 @click.option("--preview", is_flag=True, help="Allow preview advice")
-@click.option("--only", default=".*", help="Filter advice names by regex")
+@click.option("-a", "--all", is_flag=True, help="Use all advice")
+@click.argument("advice_names", nargs=-1)
+# Modifying how fixing happens
 @click.option("-n", "--dry-run", is_flag=True)
+@click.option("--target", default=".", help="Path to check, defaulting to current dir")
+@click.option(
+    "--in-branches", is_flag=True, help="In independent branches instead of inplace"
+)
+# Special non-fixing modes
+@click.option("--selftest", is_flag=True, help="Check viability of advice repo")
+@click.option("--config", is_flag=True, help="Show configuration")
+@click.pass_context
 def main(
     ctx: click.Context,
+    # Logging config
     v: Optional[int],
     vmodule: Optional[str],
+    # Advice dir
     advice_url: str,
     advice_dir: Optional[str],
     skip_update: bool,
+    # Choice of advice
     confidence: str,
     preview: bool,
-    only: str,
     dry_run: bool,
+    target: str,
+    in_branches: bool,
+    all: bool,
+    advice_names: List[str],
+    # Operation
+    selftest: bool,
+    config: bool,
 ) -> None:
     vmodule_init(v, vmodule)
     if advice_dir is None:
         advice_path = update_local_cache(advice_url, skip_update)
     else:
         advice_path = Path(advice_dir)
 
+    if config and selftest:
+        raise Exception("Can't enable both --config and --selftest simultaneously")
+
+    if advice_names:
```

Comment:
Reject if `-a` and `advice_names` are both supplied.

Reply:
Done.